frontend/src/app/shared/pop-up-window/popup-service/popup.service.ts

```diff
@@ -0,0 +1,13 @@
+import { Injectable } from '@angular/core';
+import { Router } from '@angular/router';
+
+@Injectable()
+export class PopupService {
+
+  constructor() { }
+
+  displayMessage(message: string, router: Router) {
```

Comment:
Rewrite this method in the future

Reply:
Already changed. Now parameter passed through url without changing it